### PR TITLE
[docs] Migrate new features and ideas

### LINKED ISSUE #173: [docs] Migrate New_feature_ideas
### The URL of the page on https://docs.moodle.org/dev/New_feature_ideas

New_feature_ideas

### Is this documentation specific to a Moodle version?

No

### What location would you suggest in the new docs?

I think this information should be placed as a section in the general/development/new-features-ideas.md (probably will need to adapt part of the text to avoid repetitions).

### PR DESCRIPTION
**Pull request by laurentdavid**
Migrated pages :

* https://docs.moodle.org/dev/New_feature_ideas

Closes #173 

<a href="https://gitpod.io/#https://github.com/moodle/devdocs/pull/200"></a>

### CODE REVIEW

File: general/community/contribute/new-features-ideas.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: New features and ideas
+tags:
+- Core development
+- Processes
+- Community
```
Comment (andrewnicols) [inline general/community/contribute/new-features-ideas.md:6]: Suggest adding some more tags, for example:

- Ideas
- Getting started

File: general/community/contribute/new-features-ideas.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+## Where to begin
+
+- Start by [searching moodle.org](http://moodle.org/public/search/) to check whether someone else has had the same idea. The Moodle community is very large, so it's quite likely you'll find someone else with the same idea as you ;-)
+- Either join an existing discussion about your idea or start a new discussion in an appropriate [forum on moodle.org](http://moodle.org/course/view.php?id=5).
```
Comment (andrewnicols) [inline general/community/contribute/new-features-ideas.md:12]: Suggest moving the link, and removing 'Either'

```suggestion
- Join an existing discussion about your idea, or start a new discussion in an [appropriate forum on moodle.org](http://moodle.org/course/view.php?id=5).
```

File: general/community/contribute/new-features-ideas.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## How to maximize the chance of your idea being implemented
+
+Ideas with lots of votes are more likely to be implemented, however votes are not the only deciding factor.
+
+- If you're able to provide funding for your idea to be implemented, please contact a [Moodle Partner specializing in custom development](http://moodle.com/partners/).
+- Alternatively, you could join the [Moodle Users Association](https://moodleassociation.org/) and propose your idea there.
+- If you're a developer and can create a patch for your feature idea or improvement,
+  please attach it to the tracker issue for review by the component maintainer. When you add a patch, add a "patch" label so your patch will be found in searches.
+  The M in Moodle stands for modular, and so the easiest ideas for new developers to implement are new modules and plugins.
```
Comment (andrewnicols) [inline general/community/contribute/new-features-ideas.md:30]: Take this line out - it's not true. Moodle has not been an acronym for 15 years. Perhaps re-word it?

File: sidebars/general.js [hunk 4]
```diff
@@ -26,7 +26,21 @@ const sidebars = {
     community: [
         'community/intro',
         'community/mission',
-        'community/contribute',
+        {
+            label: 'Contributing',
+            type: 'category',
+            items: [
+                {
+                    label: 'New ideas',
```
Comment (andrewnicols) [inline sidebars/general.js:34]: Not sure about the naming for this. Maybe 'Suggesting features'

**Review by andrewnicols — CHANGES_REQUESTED** [review]
Just a couple of minor things

**Comment by andrewnicols** [comment]
I'm converting this back to a draft because we discussed that it may be better to merge this into another page as this one does not really offer very much.

**Review by sarjona — APPROVED** [review]
Hi @laurentdavid!
Thanks for helping with the migration of this page. I agree that merging it with the current Contributing page makes sense and, although I'm not sure if having all the information as an admonition is the best way to display it, I'm happy to merge it and see what other people think about it :-)

**Pull request merged**